README.md

```diff
@@ -260,6 +260,7 @@ Table of Contents
   * [Braid](https://www.braidchat.com/) — Chat app designed for teams. Free for public access group, unlimited users, history, and integrations. also it provide self-hostable open-source version.
   * [cally.com](https://cally.com/) — Find the perfect time and date for a meeting. Simple to use, works great for small and large groups.
   * [Discord](https://discordapp.com/) — Chat with public/private Rooms & VoIP Service. Free for unlimited users.
+  * [Edworking](https://edworking.com/) — Project manager + Collaboration tool + Freelancers for free. The virtual office where the world’s workforce can meet, chat and collaborate, with first-class freelancers standing by to power your projects.
```

Comment:
While yeah, it says FREE all over the place, I am wondering what the details are here? Their signup shows basically the same for Team as for Freelancer, I guess you only get to discover what the limitations are post signup.

How do I prove I am a freelancer? Do I need to send anything and someone verify it? There's really not a lot of details here so I am reluctant to list something thats not up front about things